packages/component/package.json

```diff
@@ -66,7 +66,7 @@
     "react-dictate-button": "1.2.2",
     "react-film": "2.0.2",
     "react-redux": "7.2.0",
-    "react-say": "2.0.0",
+    "react-say": "2.0.1-master.c5d6304",
```

Comment:
PR needs to be reviewed.

Reply:
Here is the diff. I will include it in this PR top conversation for visibility.

https://github.com/compulim/react-say/compare/v2.0.0..c5d6304aea25aef665e6f735a5376e0b60222c4f

Reply:
@beyackle if you have the time, would you mind looking at the code above? This is a package we use in Web Chat, and those changes also need to be reviewed.

Reply:
Just one comment here - it looks like the only time you pass two arguments to `createCustomEvent` is when you're giving it an error message. So the whole `Object.entries` loop is never getting called, unless this is planned to be used elsewhere that I'm just not aware of.

Reply:
@compulim can you by chance add @beyackle to this repo for review? And/or the PR is fine.